docs/src/statmodels.md

```diff
@@ -1,6 +1,7 @@
 # Abstraction for Statistical Models
 
-This package defines an abstract type `StatisticalModel`, and an abstract subtype `RegressionModel`.
+[StatsAPI.jl](https://github.com/JuliaStats/StatsAPI.jl) defines an abstract type `StatisticalModel`,
+and an abstract subtype `RegressionModel`. They are both extented by StatsBase, and documented here.
```

Comment:
I wouldn't mention that StatsBase extends these types since that's the case only for a pair of functions (which should probably be implemented in StatsAPI too):
```suggestion
[StatsAPI.jl](https://github.com/JuliaStats/StatsAPI.jl) defines an abstract type `StatisticalModel`,
and an abstract subtype `RegressionModel`. Functions defined for these types in StatsAPI
are reexported by StatsBase and documented here for convenience.
```

Reply:
I was wondering about moving those as well, since we have the others now...